nix/default.nix

```diff
@@ -14,6 +19,35 @@ let
   python = import ./requirements.nix { inherit pkgs; };
   version = builtins.replaceStrings ["\n"] [""]
     (builtins.readFile (toString ../version.txt));
+  scriptWorkerConfig = pkgs.writeTextDir "scriptworker.yaml" (builtins.readFile ./scriptworker.yaml);
+  workerConfigs = ./configs;
+  cmd = pkgs.writeScriptBin "generateconfig" ''
```

Comment:
maybe also renaming this script would make sense, since we are also starting the shipitscript

Reply:
Yeah, makes sense.